### PR TITLE
Alles Clara

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: clara/index.html [hunk 1]
```diff
@@ -43,42 +43,42 @@ <h2 class="nodisplay">Clara</h2>
 
 	<div class="seven columns abstandU flyingintro">
 		<img class="claralogo" src="alles-clara_logo.svg" alt="ClaraLogo">
-		<p><strong>platform for caregiving relatives</strong></p>
-		<p><span class="sblcofo">UX Lead & Co-Productowner (2020 - 2022)</span></p>
+		<p><strong>A Platform for Caregiving Relatives</strong></p>
+		<p><span class="sblcofo">UX Lead and Co-Product Owner (2020 – 2022)</span></p>
```
Comment (deutschbitte) [inline clara/index.html:47]: Are you still working on this? The text says you're now only UX but the "end date" implies you stopped working on it this year. If you're still on the team, we can change this to "present"

File: clara/index.html [hunk 2]
```diff
@@ -43,42 +43,42 @@ <h2 class="nodisplay">Clara</h2>
...
+	<div class="six columns"><p>Around 1 million people in Austria look after family members or other people close to them. They provide support, organize visits to the doctor, help with eating, or are simply present.</p>
+		<p>However, this often leaves caregivers with more questions than answers. So Alles Clara provides professional advisers and online counselors to help answer questions and give recommendations and assistance.</p></div>
 	<div class="six colums"><img src="clara_illu2.png"></div>
 	<div class="twelve columns">		
-		<h3>Product strategy & management</h3>
+		<h3>Product Strategy and Management</h3>
 	</div>	
 	<div class="six columns">
-		<p>I helped to establish the initial vision and to push things forward in my role as Co-Productowner:
+		<p>I helped establish the initial vision of the app and to push things forward in my role as Co-Product Owner by:
```
Comment (deutschbitte) [inline clara/index.html:64]: This is another use of "push things forward." What specifically do you mean by this here? You helped drive the development or evolve the project, or?

"I helped establish the initial vision of the app and to push things forward in my role as Co-Product Owner by:"

File: clara/index.html [hunk 3]
```diff
@@ -43,42 +43,42 @@ <h2 class="nodisplay">Clara</h2>
...
-				<li>defining vision and roadmap</li>
-				<li>exploring and defining features</li>
-				<li>organising our team's work in meetings, on Miro & Slack</li>
-				<li>co-work with developers to translate mocks and flows to code and interactivity</li>
-				<li>test, find & help fixing those bugs</li>
-				<li>organise - run - analyse user tests</li>
+				<li>Defining the vision and roadmap</li>
+				<li>Exploring and defining features</li>
+				<li>Organizing our team's work in meetings, on Miro, and on Slack</li>
+				<li>Working with developers to translate mockups and flows to code and interactivity</li>
```
Comment (deutschbitte) [inline clara/index.html:69]: What do you mean by "translate to...interactivity?"

"Working with developers to translate mockups and flows to code and interactivity"

File: clara/index.html [hunk 4]
```diff
@@ -87,19 +87,19 @@ <h3>Clara design system</h3>
 	<div class="three colums"><img src="clara-ds_05.png"></div>
 	<div class="three colums"><img src="clara-ds_06.png"></div>
 	<div class="twelve columns">
-	<h3>Clara UX</h3>
-	<p>The platform and apps are currently in active development. We recently finished the initial alpha versions for the client app (Web / iOS / Android app for the caregiving relatives) and consultant app (Web UI for professionell consultation work). The alpha version was meant for small scale user tests and to learn and further develop the product. We are now gearing towards a bete release with invited users in 2022.</p>	
+	<h3>UX</h3>
+	<p>The platform and apps are currently in active development. We recently finished the initial alpha versions for the client app (Web / iOS / Android app for caregiving relatives) and consultant app (Web UI for professional consultation work). The alpha version was meant for small-scale user tests and to learn and further develop the product. We're now gearing toward a beta release with invited users in 2022.</p>
```
Comment (deutschbitte) [inline clara/index.html:91]: Has this already happened?

"We're now gearing toward a beta release with invited users in 2022."

File: clara/index.html [hunk 5]
```diff
@@ -87,19 +87,19 @@ <h3>Clara design system</h3>
...
+	<h3>Crew</h3>	
+	<p>The initiator and incubator is <strong><a href="https://www.two-next.com/">Two Next GmbH</a></strong>, which has Nicole Traxler as Managing Director. There are a handful of major <strong><a href="https://alles-clara.at/#partner">Austrian non-profits as project partners</a></strong>. The technical side is driven by the <strong><a href="https://www.semanticlabs.at/">Semantic Labs</a></strong> team. I was brought on during the initial research phase to help with design thinking, exploration, and the initial product definition. Once "Clara" was born as a proper vision, we moved forward and started working on the MVP. Over time, this also meant I could expand the design work to collaborate with long-term partners and friends of mine. Therefore, the Clara design credits now include: </p>
```
Comment (deutschbitte) [inline clara/index.html:97]: Does this mean the people listed below are these long-term partners and friends?

"Over time, this also meant I could expand the design work to collaborate with long-term partners and friends of mine. Therefore, the Clara design credits now include:"

File: clara/index.html [hunk 6]
```diff
@@ -87,19 +87,19 @@ <h3>Clara design system</h3>
...
 		<p><ul>
-			<li><a href="http://www.smeidu.com/">Markus Schmeiduch</a>: Design & UX lead, Co-Productowner</li>
-			<li><a href="?">Robin Westerhoff</a>: Visual design, Design system</li>						
-			<li><a href="?">Justus Wunschik</a>: Print design</li>				
-			<li><a href="?">Anna Frey</a>: Illustrations</li>											
+			<li><a href="http://www.smeidu.com/">Markus Schmeiduch</a> — Design and UX Lead, Co-Product Owner</li>
+			<li><a href="?">Robin Westerhoff</a> — Visual Design, Design System</li>
```
Comment (deutschbitte) [inline clara/index.html:100]: These links all have ?. Do you have websites or links to add or should we remove these?

File: clara/index.html (hunk 1 above)
Comment (smeidu) [reply clara/index.html:47]: I am still working on this project, but it's already defined that I will stop working and hand over things by October 2022. We can already write it that way.

File: clara/index.html (hunk 2 above)
Comment (smeidu) [reply clara/index.html:64]: being a driving force in this project, I was defining the roadmap - working with all project members on evolving that project (lots of project management with developers, with writers, with...

File: clara/index.html (hunk 3 above)
Comment (smeidu) [reply clara/index.html:69]: maybe too creative in my choise of words :-)
simply put: I defined the design (mockups and flows) and then worked with developers to create the actual functional application based on that design.

File: clara/index.html (hunk 4 above)
Comment (smeidu) [reply clara/index.html:91]: yes. that happened in June 2022.

File: clara/index.html (hunk 5 above)
Comment (smeidu) [reply clara/index.html:97]: let's skip the "friends" here. Many of my longterm freelance partner are also my personal friends, but best to mention that at all.

File: clara/index.html (hunk 1 above)
Comment (deutschbitte) [reply clara/index.html:47]: So for the text, what do you want to say here in the second sentence. It says "to fully concentrate on UX" but maybe we cut that and just say you handed duties over to the team or something. What do you think?

In 2022, after 15 months, Alles Clara had grown from a small "startup" setup to a proper full stack company effort. As a result, I handed over product owner duties to an FTE to fully concentrate on UX.

File: clara/index.html (hunk 4 above)
Comment (deutschbitte) [reply clara/index.html:91]: OK I rephrased a bit. But maybe we should give a year for what "recently" refers to in the second sentence to keep this evergreen...

"The platform and apps are currently in active development. We recently finished the initial alpha versions for the client app (Web / iOS / Android app for caregiving relatives) and consultant app (Web UI for professional consultation work). The alpha version was meant for small-scale user tests and to learn and further develop the product. In June of 2022, we invited select users to the beta release."

Comment (smeidu) [reply clara/index.html]: We finished the initial alpha version in April 2021.
In June 2022, we invited selected users to the beta version.

File: clara/index.html (hunk 6 above)
Comment (deutschbitte) [reply clara/index.html:99]: I'm merging now but a reminder to add the links here

**Pull request merged**